src/vuestic-theme/vuestic-components/va-chip/VaTag.demo.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <VbDemo>
+    <VbContainer title="Tag Types">
+      <va-tag>
+        Tag
+      </va-tag>
+      <va-tag v-model="isCloseableTagVisible" closeable>
```

Comment:
I suggest `@close="log('close')"` here. The normal case for tag is something like this

```html
<va-tag
  v-for="product in products"
  removable
  @close="removeTag(product)"
>
  {{ product.name }}
</va-tag>
```

Also `removable` is better naming than `closeable`, I reckon.